From be9379ced4d1ce451b37faaceb0a66425c708138 Mon Sep 17 00:00:00 2001
From: Glenn Washburn <development@efficientek.com>
Date: Fri, 17 Sep 2021 23:04:04 +0000
Subject: [PATCH 050/261] tests: Do not delete filesystem images on error

The filesystem images created for the filesystem test can be useful when
debugging why a filesystem test failed. So, keep them around and let the
user clean them up.

Signed-off-by: Glenn Washburn <development@efficientek.com>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
---
 tests/util/grub-fs-tester.in | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/tests/util/grub-fs-tester.in b/tests/util/grub-fs-tester.in
index ce9ae04f4..2dcd09f5e 100644
--- a/tests/util/grub-fs-tester.in
+++ b/tests/util/grub-fs-tester.in
@@ -932,9 +932,6 @@ for LOGSECSIZE in $(range "$MINLOGSECSIZE" "$MAXLOGSECSIZE" 1); do
 				sleep 1
 			    done
 			done
-			for i in $(range 0 $((NDEVICES-1)) 1); do
-			    rm "$FSIMAGEP${i}.img"
-			done
 			exit 99;
 		    fi
 		    ;;
-- 
2.36.1

